src/main/java/mate/academy/spring/model/User.java

```diff
@@ -16,6 +20,21 @@ public class User {
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany
+    @JoinTable(
+            name = "users_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles;
```

Comment:
```suggestion
    private Set<Role> roles;
```
if you are using set here don't forget also to implement equals and hashcode methods as described in common mistakes